### DISCUSSION TITLE
Is Flask 2 support over?

### DISCUSSION THREAD

**Discussion by bradbm** [pallets/flask#5358] [Q&A] [answered]
Looking for confirmation that with Flask 3 being released, when the end of life / support for Flask 2 is coming. Couldn't find any other discussion around this.

**Comment by davidism** [comment, marked as answer]
Regardless of the version number, only the latest release is supported, across all Pallets projects. We always encourage users to update as soon as they are able to.

**Reply by bradbm** [reply]
Thanks David, this is what I expected, glad to have it confirmed.
